code/modules/projectiles/ammo_datums.dm

```diff
@@ -1348,6 +1348,100 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 
 	return ..()
 
+// TE Lasers //
+
+/datum/ammo/energy/lasgun/marine
+	icon_state = "bluelaserstandard"
+	damage = 20
+	penetration = 10
+	sundering = 1
+	max_range = 40
+	damage_falloff = 0
+	bullet_color = COLOR_DISABLER_BLUE
+
+/datum/ammo/energy/lasgun/marine/overcharge
+	name = "overcharged laser bolt"
+	icon_state = "blueheavylaser"
+	hud_state = "laser_sniper"
+	damage = 40
+	penetration = 20
+	sundering = 2
+
+/datum/ammo/energy/lasgun/marine/blast
+	name = "wide range laser blast"
+	icon_state = "bluelaserstandard"
+	hud_state = "laser_spread"
+	bonus_projectiles_type = /datum/ammo/energy/lasgun/marine/spread
+	bonus_projectiles_amount = 2
+	bonus_projectiles_scatter = 5
+	accuracy_var_low = 9
+	accuracy_var_high = 9
+	accurate_range = 5
+	max_range = 8
+	damage = 35
+	penetration = 20
+	sundering = 1
+
+/datum/ammo/energy/lasgun/marine/spread
+	name = "additional laser blast"
+	icon_state = "bluelaserstandard"
+	shell_speed = 2
+	accuracy_var_low = 9
+	accuracy_var_high = 9
+	accurate_range = 5
+	max_range = 8
+	damage = 35
+	penetration = 20
+	sundering = 1
+
+/datum/ammo/energy/lasgun/marine/autolaser
+	damage = 15
+	penetration = 15
+
+/datum/ammo/energy/lasgun/marine/sniper
+	name = "sniper laser bolt"
+	icon_state = "blueheavylaser"
+	hud_state = "laser_sniper"
+	damage = 60
+	penetration = 30
+	sundering = 4
+
+/datum/ammo/energy/lasgun/marine/sniper_heat
+	name = "sniper heat bolt"
+	icon_state = "bluemicrowavelaser"
+	hud_state = "laser_heat"
+	damage = 40 //requires mod with -0.15 multiplier should math out to 10
+	penetration = 0 // It's a laser that burns the skin! The fire stacks go threw anyway.
+	flags_ammo_behavior = AMMO_ENERGY|AMMO_INCENDIARY|AMMO_SUNDERING
+	sundering = 1
+
+
+/datum/ammo/energy/lasgun/marine/pistol
+	damage = 25
+	penetration = 5
+
+/datum/ammo/energy/lasgun/marine/pistol/disabler
+	name = "disabler bolt"
+	icon_state = "disablershot"
+	hud_state = "laser_disabler"
+	damage = 70
+	penetration = 0
+	damage_type = STAMINA
+
+/datum/ammo/energy/lasgun/marine/pistol/heat
```

Comment:
i'd rather give the pistol some burst fire mode

Reply:
Don't see a need.
It uses so much charge, you only get 4 shots if you use heat only.